dashboard.el

```diff
@@ -62,6 +62,11 @@
   "Settings that are used in the Dashboard"
   :group 'dashboard)
 
+(defcustom dashboard-center-content t
```

Comment:
I would rather have the default value as nil (non-centered)... might need to update the documentation with this change.